pkg/mkimage-raw-efi/install

```diff
@@ -25,8 +25,14 @@ BAIL_FINAL_CMD=${BAIL_FINAL_CMD:-"exit 1"}
 [ -n "$DEBUG" ] && set -x
 
 pause() {
+   echo "stop rungetty.sh to not re-run login"
+   killall -STOP rungetty.sh
+   echo "killing login"
+   killall login
    echo "Pausing before $1. Entering shell. Type 'exit' to proceed to $1"
-   sh
+   sh </dev/console >/dev/console 2>&1
```

Comment:
Well, `/dev/console` is our [expectation](https://github.com/lf-edge/eve/blob/9635165c198faae1f44a84b946fb983726f555ac/pkg/mkimage-raw-efi/config.json#L9). If it is configured in wrong way we should adjust `console=` options.

What I want to have here is the way to run some general program and make it accessible from console. It may be sh now, but we may have another interactive program in future. So I am not sure that just waiting will be enough here.

Reply:
Ok, I guess it is fine to dedicate one TTY for the installer. That doesn't mean though that we need to kill all other TTY's. User specified multiple `console=` for a reason.

A better way would be not starting `getty` on this TTY on the first place. Or run install in `getty`context.